app/emoji-emojione/lib/generateEmojiIndex.mjs

```diff
@@ -1,5 +1,7 @@
 /* eslint-disable */
 
+// before using this script make sure to run: npm i --no-save node-sprite-generator
+
 // node --experimental-modules generateEmojiIndex.mjs
 import fs from 'fs';
 import nsg from 'node-sprite-generator';
```

Comment:
Did you mean to remove this?

Reply:
I don't want to remove the script itself, but maybe it is a good idea to move it to `.scripts` folder, since it is not part of the rocket.chat system :thinking: